lib/galaxy/model/__init__.py

```diff
@@ -5226,6 +5228,7 @@ def get_converted_dataset(self, trans, target_ext, target_context=None, history=
                     deps=deps,
```

Comment:
Should we pass `use_cached_job=use_cached_job` also in the recursive call at line 5206?